gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -478,7 +478,7 @@ gst_tensor_converter_set_property (GObject * object, guint prop_id,
         break;
       }
 
-      if (g_ascii_strcasecmp (strv[0], "custom") == 0)
+      if (g_ascii_strcasecmp (strv[0], "custom-code") == 0)
         self->mode = _CONVERTER_MODE_CUSTOM;
```

Comment:
Don't need to change the corresponding enum?

Reply:
Updated.